src/meter/__tests__/Meter.test.tsx

```diff
@@ -60,7 +60,7 @@ describe("Meter", () => {
     expect(meter).toHaveAttribute("aria-valuemin", "0");
     expect(meter).toHaveAttribute("aria-valuemax", "1");
     expect(meter).toHaveAttribute("aria-valuenow", "0");
-    expect(meter).toHaveAttribute("aria-valuetext", "0%");
+    expect(meter).toHaveAttribute("aria-valuetext", "0");
```

Comment:
So at first it was without % then we changed it to have % and now we removed it? any particular reason for this? how does it effects the voice over?

Reply:
I does affects VO, yesterday I was seeing Progress and blindly changed for both meter & progress in a hurry.

Progress announces percent automatically with the value text.
But for meter it doesn't announces, thats the reason I added before.

I need to revert them back.